tests/test_sap.py

```diff
@@ -563,3 +563,139 @@ def test_generate_sap_data_success():
  \
 \n"
     )
+
+def test_generate_summary():
+    invoices = [
```

Comment:
This should probably be a fixture given the size, your thoughts @hakbailey ?

Reply:
Yeah, I would either create a new fixture or update the existing invoices fixture to include the credit card payment invoice (that would mean updating some other impacted tests though).